stl/src/syserror.cpp

```diff
@@ -113,29 +112,59 @@ _CRTIMP2_PURE unsigned long __CLRCALL_PURE_OR_CDECL _Winerror_message(
     unsigned long _Message_id, char* _Narrow, unsigned long _Size) { // convert to name of Windows error, return 0 for

```

Comment:
Preexisting: insert newline after the { to fix `clang-format` wrapping.

Reply:
Not sure what `clang-format` wrapping is to fix here.